Kconfig.zephyr

```diff
@@ -922,3 +922,10 @@ config LEGACY_INCLUDE_PATH
 	  but enabling CONFIG_LEGACY_INCLUDE_PATH will allow developers to use
 	  <kernel.h> instead. This (without the zephyr/ prefix) is deprecated
 	  and should be avoided. Eventually, it will not be supported.
+
+config GENERATE_VIF
```

Comment:
This kconfig option is used in cmake to decide wether to execute the script during build, it will be set to y in projects that needs the XML file in their build hence kept it at top level

Reply:
Moved this config option to suggested folder